tools/toolchains/arm.py

```diff
@@ -285,6 +285,9 @@ def __init__(self, target, *args, **kwargs):
         elif target.core.lower().endswith("f"):
             self.flags['common'].append("-mcpu=%s" % target.core.lower()[:-1])
             self.flags['ld'].append("--cpu=%s" % target.core.lower()[:-1])
+        elif target.core.lower().endswith("ns"):
```

Comment:
what does `ns` at the end of the core signify? NEON?

Reply:
NS - Non Secure

Reply:
Thanks!